tests/python/pytorch/dataloading/test_dataloader.py

```diff
@@ -622,6 +622,36 @@ def test_edge_dataloader_excludes(
             break
 
 
+def test_edge_dataloader_exclusion_without_all_reverses():
+    data_dict = {('A','AB','B'): (torch.tensor([0,1]),torch.tensor([0,1])),
+             ('B','BA','A'): (torch.tensor([0,1]),torch.tensor([0,1])),
+            ('B','BC','C'):(torch.tensor([0]),torch.tensor([0])),
+            ('C','CA','A'): (torch.tensor([0,1]),torch.tensor([0,1]))}
+    g = dgl.heterograph(data_dict=data_dict)
+    block_sampler = dgl.dataloading.MultiLayerNeighborSampler(
+                fanouts=[1],
+                replace=True)
+    block_sampler = dgl.dataloading.as_edge_prediction_sampler(
+                block_sampler,
+                exclude="reverse_types",
+                reverse_etypes={'AB':'BA'},
+
+            )
+    d = dgl.dataloading.DataLoader(
+                graph=g,
+                indices={'AB':torch.tensor([0]),'BC':torch.tensor([0]),},
+                graph_sampler=block_sampler,
+                batch_size=2,
+                shuffle=True,
+                drop_last=False,
+                num_workers=0,
+                device='cuda',
```

Comment:
The test is for CUDA only. My understanding is that this change applies to both CPU and GPU, so consider set the device according to the global context.